sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -15336,4 +15336,36 @@ public void testFilterParseLongNullable()
         ImmutableList.of(new Object[]{NullHandling.sqlCompatible() ? 4L : 0L})
     );
   }
+
+  @Test
+  public void testLatestByAggregatorOnSecondaryTimestamp()
```

Comment:
this test passes even without the changes in this PR, am i missing something?

Reply:
This test was there to show the use of a constant value but that is not a value addition. I have updated it